docker/Dockerfile

```diff
@@ -139,6 +139,9 @@ RUN apt-get update && \
 ARG CHECKPOINT_SYNC=true
 ARG NETWORK=Mainnet
 
+ARG ZEBRA_CONF_PATH=/etc/zebrad/zebrad.toml
+ENV ZEBRA_CONF_PATH ${ZEBRA_CONF_PATH}
```

Comment:
Can you add this environmental variable to the Docker user documentation?

Reply:
Where's that documentation? 😅

Reply:
Nevermind. Added this for tracking https://github.com/ZcashFoundation/zebra/issues/5167